tests/integration/study_data_blueprint/test_table_mode.py

```diff
@@ -945,3 +858,207 @@ def test_table_type_aliases(client: TestClient, user_access_token: str) -> None:
+            },
+            "es / fr": {
+                "colorr": 200,
+                "colorg": 150,
+                "colorb": 100,
+                "displayComments": True,
+                "hurdlesCost": True,
+                "linkStyle": "plain",
+                "linkWidth": 1,
+                "loopFlow": False,
+                "transmissionCapacities": "enabled",
+                "usePhaseShifter": True,
+            },
+            "fr / it": {
+                "assetType": "DC",  # case-insensitive
+            },
+        }
+
+        expected_links = {
+            "de / fr": {
+                "area1": "de",
+                "area2": "fr",
+                "assetType": "ac",
+                "colorb": 100,
+                "colorg": 150,
+                "colorr": 200,
+                "displayComments": False,
+                "hurdlesCost": True,
+                "linkStyle": "plain",
+                "linkWidth": 2,
+                "loopFlow": False,
+                "transmissionCapacities": "ignore",
+                "usePhaseShifter": False,
+            },
+            "de / it": {
+                "area1": "de",
+                "area2": "it",
+                "assetType": "ac",
+                "colorr": 112,
+                "colorg": 112,
+                "colorb": 112,
+                "displayComments": True,
+                "hurdlesCost": False,
+                "linkStyle": "plain",
+                "linkWidth": 1,
+                "loopFlow": False,
+                "transmissionCapacities": "enabled",
+                "usePhaseShifter": False,
+            },
+            "es / fr": {
+                "area1": "es",
+                "area2": "fr",
+                "assetType": "ac",
+                "colorb": 100,
+                "colorg": 150,
+                "colorr": 200,
+                "displayComments": True,
+                "hurdlesCost": True,
+                "linkStyle": "plain",
+                "linkWidth": 1,
+                "loopFlow": False,
+                "transmissionCapacities": "enabled",
+                "usePhaseShifter": True,
+            },
+            "fr / it": {
+                "area1": "fr",
+                "area2": "it",
+                "assetType": "dc",
+                "colorb": 112,
+                "colorg": 112,
+                "colorr": 112,
+                "displayComments": True,
+                "hurdlesCost": True,
+                "linkStyle": "plain",
+                "linkWidth": 1,
+                "loopFlow": False,
+                "transmissionCapacities": "enabled",
+                "usePhaseShifter": False,
+            },
+        }
+    else:
+        json_input = {
+            "de / fr": {
+                "assetType": "ac",
+                "colorb": 100,
+                "colorg": 150,
+                "colorr": 200,
+                "displayComments": False,
+                "filterSynthesis": "hourly, daily, weekly, annual",
```

Comment:
Couldn't you just add these keys ?
json_input = ...
if version > 82:
   add keys.

It would be shorter and clearer

Same for the rest of the method

Reply:
Ok done